README.md

```diff
@@ -30,12 +30,26 @@ Python wrapper around C++ helper to compute, for each point in a 3D point cloud,
 - volume
 - curvature
 - optimal neighborhood size
-
 </details>
 
 
 ## 🧱 Installation
 
+```bash
```

Comment:
Could you please at some subsections to clarify the two installation modes ? 

## 🧱 Installation

### Inside an existing environment 

```bash
# clone project
git clone https://github.com/drprojects/point_geometric_features.git
cd point_geometric_features

# run 
python -m pip install .
```

Or, if you do not need the source files:

```bash
python -m pip install git+https://github.com/drprojects/point_geometric_features
```

Pgeof depends on [Eigen library](https://eigen.tuxfamily.org/) and numpy headers at build time.
The good version of Numpy will be fetched from PyPI automatically but your are responsible for providing
the path to the Eigen library you want to use (for example by using `CXXFLAGS` variable on Linux or setting `EIGEN_LIB_PATH`).

Pgeof will be soon available as pre compiled package on PyPI.

### In a new dedicated `conda` environment 

The following will install the project in a new `conda` environment named `pgeof`.

```bash
# clone project
git clone https://github.com/drprojects/point_geometric_features.git
cd point_geometric_features

# Installation in a new dedicated `pgeof` conda environment
bash install.sh
```

You can easily adapt `install.sh` to install the project in an already-existing 
environment.

Reply:
Ok!